tests/internal/test_forksafe.py

```diff
@@ -288,61 +288,91 @@ def fn():
     assert exit_code == 42
 
 
-# FIXME: subprocess marks do not respect pytest.mark.skips
-if sys.version_info < (3, 11, 0):
-
-    @pytest.mark.subprocess(
-        out=("CTCTCT" if sys.platform == "darwin" or (3,) < sys.version_info < (3, 7) else "CCCTTT"),
-        err=None,
-        env=dict(_DD_TRACE_GEVENT_HUB_PATCHED="true"),
-    )
-    def test_gevent_reinit_patch():
-        import os
-        import sys
+@pytest.mark.subprocess(
+    out=("CTCTCT" if sys.platform == "darwin" or (3,) < sys.version_info < (3, 7) else "CCCTTT"), err=None
+)
+def test_gevent_gunicorn_behaviour():
```

Comment:
I like keeping it because it provides slightly lower-level detail about why patching order matters with gunicorn than the integration test Yun and I wrote.